### DISCUSSION TITLE
开启讨论区

### DISCUSSION THREAD

**Discussion by lisai9093** [lisai9093/YYS#3] [Announcements]
请在这里讨论本脚本的问题/建议/吹水。为了保护隐私暂时不打算用其他社交软件（微信QQ等）

**Comment by XiaFanbob** [comment]
准备工作都做完了，运行脚本不会用233

**Reply by XiaFanbob** [reply]
选择功能模式是输入数字吗，输完游戏也没反应

**Reply by lisai9093** [reply]
是数字。windows下运行的话记得要用管理员模式打开IDLE

**Reply by XiaFanbob** [reply]
这个具体是怎么运行的，打开游戏，脚本。然后输入想要的模式他就自动开始了吗，还是游戏要转到特定的页面。我运行了脚本游戏没反应

**Reply by lisai9093** [reply]
每个模式需要到对应的界面

**Comment by Alex661** [comment]
想问下up的电脑分辨率和电脑字体设置大小（100%还是125%）?

**Reply by lisai9093** [reply]
默认全部是100% （无DPI），可以单独调整阴阳师的DPI这样就不用改电脑设置了。
